Dockerfile

```diff
@@ -9,20 +9,23 @@ WORKDIR /app
 
 RUN go mod download
 
+COPY ./bindings    /app/bindings
 COPY ./core        /app/core
 COPY ./cmd         /app/cmd
 COPY ./Makefile    /app/Makefile
-COPY ./bot.toml    /app/bot.toml
+COPY ./bot.testnet.toml    /app/bot.testnet.toml
+COPY ./bot.mainnet.toml    /app/bot.mainnet.toml
 
 WORKDIR /app/
 
 RUN make build-go
 
 FROM alpine:3.18
 
-COPY --from=builder /app/bot /usr/local/bin
-COPY --from=builder /app/bot.toml /app/bot.toml
+COPY --from=builder /app/bot              /usr/local/bin
+COPY --from=builder /app/bot.testnet.toml /bot.testnet.toml
+COPY --from=builder /app/bot.mainnet.toml /bot.mainnet.toml
 
 WORKDIR /app
 
-CMD ["bot", "run", "--config", "/app/bot.toml"]
+CMD ["bot", "run", "--config", "/bot.testnet.toml"]
```

Comment:
If you want to build a mainnet version, do you still need to modify this part? Maybe mainnet and testnet can be separated into two different CI builds, distinguished by branch name or tag name.

Reply:
Hi @welkin22,I removed `CMD` from Dockerfile at [5df0332](https://github.com/bnb-chain/opbnb-bridge-bot/pull/18/commits/5df033228e89db49efcb7781c30d7ec8f25d5472) . Users can run the docker via `docker run <image> bot run --config /bot.mainnet.toml` for mainnet and `docker run <image> bot run --config /bot.testnet.toml` for testnet.